build.gradle

```diff
@@ -52,6 +52,9 @@ allprojects {
                 check("PreferSafeLogger", CheckSeverity.OFF)
                 check("PreferSafeLoggingPreconditions", CheckSeverity.OFF)
                 check("PreconditionsConstantMessage", CheckSeverity.OFF)
+                // temporarily opt out of error-prone in order to upgrade checkerframework.
+                // This should be removed as soon as baseline is next upgraded within this repo.
+                enabled = false
```

Comment:
I've compiled this using a snapshot of itself, things seem good, the problem is upgrading dependencies makes our existing baseline classes (applied to this project) incompatible with the upgraded checkerframework.

Reply:
This flag has been removed by the baseline upgrade here: https://github.com/palantir/gradle-baseline/pull/2503/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7L55-L57